javascript-timers/timeout.js

```diff
@@ -0,0 +1,5 @@
+const $message = document.querySelector('.message');
+
+setTimeout(() => {
+  $message.innerHTML = `Hello There`;
```

Comment:
You shouldn't use `innerHTML`, it opens you up to security attacks. You should be using `textContent` instead